### DISCUSSION TITLE
Internal Messages Not Displaying - Blank Screen

### DISCUSSION THREAD

**Discussion by beatriz-g-amaral** [LiveHelperChat/livehelperchat#2173] [Q&A]
When I try to send or view an internal message in Live Helper Chat, the screen turns completely white, as shown in the attached image. I cannot proceed further because no interface elements are displayed.
Expected Behavior:
The internal message interface should load properly, displaying the input fields and conversation content.

Actual Behavior:
The screen turns white with no content or UI elements.

Attachment:
![image](https://github.com/user-attachments/assets/54972513-486b-487c-a469-f33b75669a0f)

Please let me know if any logs or further details are needed to resolve this issue.

**Comment by remdex** [comment]
You should see error in audit log
https://doc.livehelperchat.com/docs/debug
